projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -212,7 +212,7 @@ public function add_appearance_menu() {
 		add_menu_page( esc_attr__( 'Appearance', 'jetpack' ), __( 'Appearance', 'jetpack' ), 'switch_themes', $themes_url, null, 'dashicons-admin-appearance', 60 );
 		add_submenu_page( $themes_url, esc_attr__( 'Themes', 'jetpack' ), __( 'Themes', 'jetpack' ), 'switch_themes', 'https://wordpress.com/themes/' . $this->domain );
 
-		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
+		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() && ! has_action( 'customize_register' ) ) {
```

Comment:
Maybe we could move the `has_action` check first. The wp_is_block theme requires a filesystem lookup which is more resource intensive than checking if an action is registered.

```suggestion
		if ( ! has_action( 'customize_register' ) && function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
```